src/types/processWrapper.ts

```diff
@@ -0,0 +1,25 @@
+import { ChildProcess } from 'child_process'
+
+export default class ProcessWrapper {
```

Comment:
In a later refactor, this function can be entirely removed. It is a noop wrapper